storybook/src/elements/nuxeo-html-editor/nuxeo-html-editor.stories.js

```diff
@@ -0,0 +1,10 @@
+import { storiesOf } from '@storybook/polymer';
+import { html } from 'lit-html';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-html-editor';
+
+storiesOf('UI/nuxeo-html-editor', module).add(
+  'nuxeo-html-editor',
+  () => html`
+    <nuxeo-html-editor></nuxeo-html-editor>
```

Comment:
nit: wondering if we should add a parent div to restrict the size of the element and this way making more clear  what the element is. The way it is now, I'm afraid people might confuse the html editor bar with the storybook actions.

Reply:
good catch, added some passing and a discrete border